pipeline/frontend/metadata.go

```diff
@@ -55,19 +55,20 @@ type (
 
 	// Pipeline defines runtime metadata for a pipeline.
 	Pipeline struct {
-		Number   int64  `json:"number,omitempty"`
-		Created  int64  `json:"created,omitempty"`
-		Started  int64  `json:"started,omitempty"`
-		Finished int64  `json:"finished,omitempty"`
-		Timeout  int64  `json:"timeout,omitempty"`
-		Status   string `json:"status,omitempty"`
-		Event    string `json:"event,omitempty"`
-		Link     string `json:"link,omitempty"`
-		Target   string `json:"target,omitempty"`
-		Trusted  bool   `json:"trusted,omitempty"`
-		Commit   Commit `json:"commit,omitempty"`
-		Parent   int64  `json:"parent,omitempty"`
-		Cron     string `json:"cron,omitempty"`
+		Number   int64    `json:"number,omitempty"`
+		Created  int64    `json:"created,omitempty"`
+		Started  int64    `json:"started,omitempty"`
+		Finished int64    `json:"finished,omitempty"`
+		Timeout  int64    `json:"timeout,omitempty"`
+		Status   string   `json:"status,omitempty"`
+		Event    string   `json:"event,omitempty"`
+		Link     string   `json:"link,omitempty"`
+		Target   string   `json:"target,omitempty"`
+		Trusted  bool     `json:"trusted,omitempty"`
+		Commit   Commit   `json:"commit,omitempty"`
+		Parent   int64    `json:"parent,omitempty"`
+		Cron     string   `json:"cron,omitempty"`
+		Labels   []string `json:"labels,omitempty"`
```

Comment:
On this level they should either be called `CI_LABELS` instead of `CI_COMMIT_PULL_REQUEST_LABELS` or we need to move that property into the Commit part.

Reply:
I'm for `CI_LABELS` as we already store them in all structs internaly this way ...

Reply:
@6543 The `Labels` property was just added in this PR. What other labels do you plan to store instead of pull-request labels in there? The only other thing I can think of atm is issue labels, but we currently have no pipeline event for issues.

Reply:
@anbraten **issues** also can have labels but for now we dont support any event in that direction ...